src/app/services/actions/profile-action-creators.ts

```diff
@@ -1,7 +1,14 @@
 import { IQuery } from '../../../types/query-runner';
-import { PROFILE_IMAGE_REQUEST_SUCCESS, PROFILE_REQUEST_ERROR, PROFILE_REQUEST_SUCCESS } from '../redux-constants';
+import { PROFILE_IMAGE_REQUEST_SUCCESS, PROFILE_REQUEST_ERROR, PROFILE_REQUEST_SUCCESS, PROFILE_TYPE_SUCCESS } from '../redux-constants';
 import { authenticatedRequest, isImageResponse, parseResponse } from './query-action-creator-util';
```

Comment:
looks like the `parseResponse` function is used in multiple places. maybe it's a good time to add some typing for it :)